spec/models/card_spec.rb

```diff
@@ -7,7 +7,10 @@
     }.to raise_error
   end
 
-  let(:card) { Card.create!(original_text: "test", translated_text: "тест") }
+  let(:card) { FactoryGirl.create(:card,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.